src/dependadotnet/Program.cs

```diff
@@ -37,18 +36,12 @@ static void Main(string[] args)
                 return;
             }
 
-            var printboilerplate = true;
+            PrintBoilerPlate();

```

Comment:
I don't see a good reason for having a bool variable that is checked each time in the loop. The boilerplate is printed once. So I just called the function here, before the loop.